app/src/main/java/com/hyperether/getgoing/ui/fragment/TrackingFragment.java

```diff
@@ -401,13 +430,14 @@ public void onClick(DialogInterface paramDialogInterface, int paramInt) {
         }
     };
 
-    private final OnClickListener mButtonBackListener = v -> onAnyBackButtonPressed();
+    private final View.OnClickListener mButtonBackListener = v -> getActivity().onBackPressed(); //onAnyBackButtonPressed();
 
-    private final OnClickListener mButtonSetGoalListener = new OnClickListener() {
+    private final View.OnClickListener mButtonSetGoalListener = new View.OnClickListener() {
         @Override
         public void onClick(View v) {
-            ActivitiesFragment activitiesFragment = ActivitiesFragment.newInstance(null, OPENED_FROM_LOCATION_ACT);
-            activitiesFragment.show(getSupportFragmentManager(), "ActivitiesFragment");
+            Bundle bundle = new Bundle();
+            bundle.putInt("from", OPENED_FROM_LOCATION_ACT);
```

Comment:
@zukovicfilip do not hardcode parameters ("from"). Please create constants / strings.